Authorize.NET/Api/Contracts/V1/RequestFactoryWithSpecified.generated.org

```diff
@@ -814,6 +866,12 @@ transRetailInfoType() {
             {
             }
         }
+        public static void ContactDetailType(ContactDetailType argument) 
+        {
+            if(null != argument) 
+            {
+            }
```

Comment:
This is redundant. Add some code here or remove the if block.

Reply:
Removed the IF block.